controllers/userController.js

```diff
@@ -0,0 +1,71 @@
+import { Users } from '../models/Users.js';
+
+//create a new user
+export const createUser = async (req, res) => {
+  try {
+    const { firstName, lastName, email, password, role } = req.body;
```

Comment:
The data from the client needs to be validated. A library like `joi` can help. Same applies to updating a user.